tests/acceptance/features/apiAntivirus/antivirusMain.feature

```diff
@@ -158,3 +158,71 @@ Feature: Antivirus basic
       | dav-path-version |
       | old              |
       | new              |
+
+  Scenario: A file cannot be overwritten with a file containing a virus via public upload
+    Given the administrator has enabled DAV tech_preview
+    And as user "Alice"
+    And user "Alice" has created a public link share of folder "FOLDER" with change permissions
+    When the public uploads file "textfile.txt" from the antivirus test data folder using the new WebDAV API
+    And the public overwrites file "textfile.txt" with content "X5O!P%@AP[4\PZX54(P^)7CC)7}$EICAR-STANDARD-ANTIVIRUS-TEST-FILE!$H+H*" using the new WebDAV API
+    Then the HTTP status code should be "403"
+    And the last lines of the log file should contain log-entries containing these attributes:
+      | user | app             | method | message               |
+      | --   | files_antivirus | PUT    | Infected file deleted |
+    And the content of file "/FOLDER/textfile.txt" for user "Alice" should be "Small text file without virus."
+
+  @skip @files_primary_s3#69
```

Comment:
These tests are skipped because of this issue : https://github.com/owncloud/files_primary_s3/issues/100

Reply:
if it is skipped because of the issue https://github.com/owncloud/files_primary_s3/issues/100 then isn't the tag supposed to be `@files_primary_s3#100 instead` of `@files_primary_s3#69` 🤔 
```suggestion
  @skip @files_primary_s3-issue-100
```